ui/components/app/signature-request-original/__snapshots__/signature-request-original.test.js.snap

```diff
@@ -6,142 +6,140 @@ exports[`SignatureRequestOriginal should match snapshot 1`] = `
     class="request-signature__container"
   >
     <div
-      class="request-signature__header"
+      class="request-signature__account"
     >
       <div
-        class="request-signature__header-background"
-      />
-      <div
-        class="request-signature__header__text"
-      >
-        Signature request
-      </div>
-      <div
-        class="request-signature__header__tip-container"
-      >
-        <div
-          class="request-signature__header__tip"
-        />
-      </div>
-    </div>
-    <div
-      class="request-signature__body"
-    >
-      <div
-        class="request-signature__account-info"
+        class="box network-account-balance-header box--padding-4 box--display-flex box--flex-direction-row box--justify-content-space-between box--align-items-center"
       >
         <div
-          class="request-signature__account"
+          class="box box--display-flex box--gap-2 box--flex-direction-row box--align-items-center"
         >
           <div
-            class="request-signature__account-text"
-          >
-            Account:
-          </div>
-          <div
-            class="request-signature__account-item"
+            class="box box--display-flex box--flex-direction-row box--align-items-center"
           >
             <div
-              class="account-list-item undefined"
-              data-testid="account-list-item"
+              class=""
             >
               <div
-                class="account-list-item__top-row"
+                class="identicon"
+                style="height: 32px; width: 32px; border-radius: 16px;"
               >
                 <div
-                  class=""
+                  style="border-radius: 50px; overflow: hidden; padding: 0px; margin: 0px; width: 32px; height: 32px; display: inline-block; background: rgb(250, 58, 0);"
```

Comment:
So many inline styles are avoidable. Please check using class over this.

Reply:
Hi @jpuri 
It's an auto-generated snapshot, I can't change this.